src/libostree/ostree-repo.c

```diff
@@ -3297,6 +3297,20 @@ reload_remote_config (OstreeRepo *self, GCancellable *cancellable, GError **erro
 static gboolean
 reload_sysroot_config (OstreeRepo *self, GCancellable *cancellable, GError **error)
 {
+  {
+    g_autofree char *boot_counting_str = NULL;
+
+    (void)ot_keyfile_get_value_with_default_group_optional (self->config, "sysroot", "boot_counting", "0",
+                                                            &boot_counting_str, NULL);
+
+    if (boot_counting_str)
+      /* Ensure boot count value is in [0,5] */
```

Comment:
Hmm, doesn't seem required by the spec. Is that a systemd-boot requirement?

Reply:
well, I took the initiative here :) . I personally don't see any reasons why we should have support for more then 5 boot attempts.

I can drop this limitation if there are objections.